packages/react-native-bpk-component-calendar/src/NativeCalendar.android.js

```diff
@@ -34,27 +33,11 @@ export type Props = {
   ...$Exact<CommonProps>,
 };
 
-const createOnChangeHandler = memoize(callback => event => {
-  if (event.nativeEvent.selectedDates) {
-    const convertedDates = event.nativeEvent.selectedDates.map(dt => {
-      const utcDate = new Date(dt);
-      return new Date(
-        utcDate.getUTCFullYear(),
-        utcDate.getUTCMonth(),
-        utcDate.getUTCDate(),
-      );
-    });
-    if (callback) {
-      callback(convertedDates);
-    }
-  }
-});
-
 const parseDateToNative = date => {
   if (date) {
     // Return as unix timestamp because the only number data type that allows null
     // in the native side is Integer, and the original milliseconds will overflow it.
-    return date.getTime() / 1000;
+    return new Date(date).getTime() / 1000;
```

Comment:
This tripped me up since `new Date(date)` when `date` is a `Date` is redundant. Can we add a comment to document that?

Reply:
Isn't the `new Date(date)` also required for IOS given that we now also support date timestamps?

Reply:
It doesn't seem to be. I think the native handler can deal with a number

Reply:
`new Date(date)` is required when `typeof date === "number"`. When it's a `Date` this is a NOP I believe, but it's a bit confusing.

![image](https://user-images.githubusercontent.com/1333960/52069629-0531da80-2577-11e9-87f2-7e7d3c2de880.png)